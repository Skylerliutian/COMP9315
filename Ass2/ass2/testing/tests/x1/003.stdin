# Create a Bits struct
N A 10
# Set the 5th least sig bit to 1
s A 5
# Print the Bits struct
O A
# Free the Bits struct
F A

# create a Bits struct
N B 100
# Set all bits to 1
S B
# Print the Bits struct
O B
# Free the Bits struct
F B