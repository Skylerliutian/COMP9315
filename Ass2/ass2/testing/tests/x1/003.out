0000000000100000
11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
